terraform/variables.tf

```diff
@@ -136,3 +136,11 @@ variable "data_snapshot_aws_secret_key" {
 variable "datadog_api_key" {
   sensitive = true
 }
+
+variable "AWS_SECRET_ACCESS_KEY" {
+  default = ""
+}
+
+variable "AWS_ACCESS_KEY_ID" {
+  default = ""
+}
```

Comment:
Necessary for local `terraform` to import/plan/apply, or is this just cleaning up the warning we currently get in all our plans in Terraform Cloud?

Reply:
Correct, this is just to clean up that warning which was preventing my local planning